air-quality-ui/system_tests/utils/graph_aqi_mock.ts

```diff
@@ -1107,3 +1107,3224 @@ export const apiForecastAqi = [
     },
   },
 ]
+
+export const mockMeasurementsObject = [
+  {
+    measurement_date: '2024-07-01T01:00:00Z',
+    location_type: 'city',
+    location_name: 'Rio de Janeiro',
+    api_source: 'OpenAQ',
+    o3: 55.63,
+    pm10: 27,
+    entity: 'Governmental Organization',
+    sensor_type: 'reference grade',
+    site_name: 'Centro',
+  },
+  {
```

Comment:
I think there will be a more concise way to generate this data using a method to make this file smaller

Reply:
Chat with devs about reducing size of file. Add clarity for each mock.